updated cwltool version, added missing download ref param

junjun-zhang opened the pull request.

junjun-zhang committed:
updated cwltool version, added missing download ref param

denis-yuen reviewed (dismissed):
Looks like parameter is in the example `Dockstore_cwl.json` already.
~Pending build success, I don't think the Java version matters at all~
Spoke too soon, I guess this tests the build of the seqware workflow

denis-yuen committed:
Try to fix build

denis-yuen committed:
Update .travis.yml

denis-yuen committed:
Looks like junction plugin is gone.

Luckily, we might not be using it
https://issues.sonatype.org/browse/MVNCENTRAL-1365

denis-yuen reviewed (commented):
I've probably made too many changes to approve this.
But it looks like its building and testing properly now
